src/lib/core/toolbar-menu/toolbar-menu.component.scss

```diff
@@ -23,7 +23,7 @@
     .blui-toolbar-icon-wrapper .mat-icon {
         font-size: 1rem;
         width: 1rem;
-        line-height: 1rem;
+        line-height: 1.3rem;
```

Comment:
How did you calculate this value?

It seems really close to being centered, but I'm curious about the exact math here to ensure it's centered.

Here are some screenshots with center lines added. It's difficult to tell exactly... could you take a look @huayunh and let me know how to test something like this in the future?
![image](https://user-images.githubusercontent.com/13989985/223470539-77db8586-b7c9-4ecd-b4ff-c2feb1f8a2df.png)

Reply:
you are doing what I would do. Center lining... inspector... My eyes can detect as low as a 1px shift, and when I eyeballed the screenshot in the description it looked good to me.

But I would try to avoid magic numbers like these unless absolutely needed